Source/MercurialSourceControl/Private/MercurialSourceControlFileState.h

```diff
@@ -76,6 +76,12 @@ class FFileState
 		History = InFileRevisions;
 	}
 
+	// kill ambiguous error message:
```

Comment:
I didn't pull in this change as I didn't see any compiler error relating to this.